app/views/assignments/_row.html.erb

```diff
@@ -23,17 +23,6 @@
       <% result = @a_id_results[assignment.id] %>
       <% if !result.nil? %>
         <% if assignment.max_mark > 0 && !assignment.results_average.nil? %>
-          <strong>
-            <%= t('results.your_mark',
-                  mark: number_to_percentage(result.total_mark * 100 / assignment.max_mark, precision: 1)) %>
-          </strong>
-          <br>
-          <%= "#{t(:class_average)}: #{number_to_percentage(assignment.results_average, precision: 1)}" %>
-          <% if assignment.display_median_to_students %>
-            <br>
-            <%= "#{t(:class_median)}: #{number_to_percentage(assignment.results_median, precision: 1)}" %>
```

Comment:
@david-yz-liu Will any reference to the median be lost to students now? Is that the direction we want to move in?

Reply:
Oh right I was thinking about it this morning and forgot to mention it. I think the right thing to do (when the median is supposed to be shown) is pass the assessment medians as more data to the bar chart, using a third bar colour for those.